module/VuFind/src/VuFind/Recommend/LibGuidesProfile.php

```diff
@@ -62,6 +62,34 @@ class LibGuidesProfile implements
      */
     protected $libGuides;
 
+    /**
+     * List of strategies enabled to find a matching LibGuides profile
+     *
+     * @var int
+     */
+    protected $strategies = [];
+
+    /**
+     * Map of call number pattern to config alias
+     *
+     * @var array
+     */
+    protected $callNumberToAlias;
```

Comment:
Rather than mapping call numbers to aliases, would it be better to map call numbers to accounts, with aliases as an optional feature? I.e. for each value, look it up in the alias table to map it to an account number, but if it's not in the alias table, assume it is _already_ an account number. This would save people from having to create aliases for accounts that are only used once or twice. (I realize that the aliases are probably going to be more useful/readable in the majority of cases, but it seems unnecessary to make them mandatory).

Reply:
Yeah I considered that.  But the real benefit to aliases (which you couldn't know since you don't have LibGuides) is that the account IDs are purely numeric, and 4-6 digits(!), making them completely inscrutable.  So even if every single call number prefix maps to a different account, I don't realistically see anyone wanting their configuration file to be unreadable enough to skip the aliases.

Reply:
Yeah if someone actually complains about this I am happy to make aliases skippable, but I don't see that happening.  I also think they may be useful for other logic in LibGuidesAPI.ini in the future.